data/common.yaml

```diff
@@ -95,6 +95,12 @@ mcollective::required_directories:
   - /etc/puppetlabs/mcollective
   - /opt/puppetlabs/mcollective
 
+mcollective::service_ensure: stopped
+mcollective::service_enable: false
+mcollective::manage_bin_symlinks: true
+mcollective::plugin_classes:
+  - choria
```

Comment:
I think these 2 lines can go - they are already in this file

Reply:
done